courses/security/fundamentals.md

```diff
@@ -42,7 +42,7 @@
   - Fail securely
     - Applications regularly fail to process transactions for many reasons. How they fail can determine if an application is secure or not.
 
-    ```
+    ```java
```

Comment:
@stormic-nomad-nishant can you confirm this? otherwise LGTM

Reply:
LGTM